packages/pluggableWidgets/datagrid-web/src/components/ColumnSelector.tsx

```diff
@@ -83,6 +83,7 @@ export function ColumnSelector(props: ColumnSelectorProps): ReactElement {
                             checked={isVisible}
                             disabled={isVisible && props.columns.length - props.hiddenColumns.length === 1}
                             id={`${props.id}_checkbox_toggle_${index}`}
+                            style={{ pointerEvents: "none" }}
```

Comment:
is this expected behaviour? maybe we need some checks or something like that?

Reply:
As the li outside controls the events i would say yes. Same is applied to the label in line 90.